src/test/java/api/UserApi.java

```diff
@@ -0,0 +1,31 @@
+package api;
+
+import com.github.javafaker.Faker;
+import io.restassured.response.Response;
+
+public class UserApi extends ApiBase {
+    Response response;
+    RegistrationDTO registrationDTO;
+    Faker faker = new Faker();
+
+    public UserApi() {
+        registrationDTO = new RegistrationDTO();
+    }
+    public RegistrationDTO randomDataForNewUser(){
+        registrationDTO.setName(faker.name().username());
```

Comment:
random data will the better if you transfer in variable under this line Faker faker = new Faker();